lms/djangoapps/course_goals/tests/test_goals.py

```diff
@@ -0,0 +1,128 @@
+"""
```

Comment:
the existing goals tests are in test_api.py so that file can be removed in the clean up ticket

Reply:
These tests are also for the API though yeah? Feels like `test_api` is a reasonable place for them - long term I kind of prefer that name to the more generic `test_goals`.

Reply:
Ah yeah I hear you that these are testing the rest endpoint. I'd also call that an API, but yeah I get you, that if the other one is testing `api.py`, you'd want some separation.

But you mention deleting `test_api.py` at some point - we'd want to keep some of those tests around still, yeah? In the brave new world, would we be able to remove all other calls into this djangoapp from platform?

Reply:
no, i replicated the tests in test_api.py in test_goals.py but for the new goals, all those tests are testing the old goals so they can all be deleted (separately from the question of what a good name for the test_goals.py file is)

Reply:
Ah yes I see the old `test_api.py` is also testing the REST API stuff. OK, I misunderstood your comment above - I guess you were saying that you only like `test_api.py` as a name for tests against `api.py`, not for REST API tests. Which doesn't personally bother me, but :shrug: who cares. Your plan is good.